Modix/Modules/NoteModule.cs

```diff
@@ -0,0 +1,200 @@
+﻿namespace Modix.Modules
+{
+    using System;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Discord;
+    using Discord.Commands;
+    using Serilog;
+    using Services.CommandHelp;
+    using Services.Notes;
+    using Tababular;
+
+    [Name("Add/Remove/Search Admin notes"), RequireUserPermission(GuildPermission.Administrator), HiddenFromHelp]
+    public class NoteModule : ModuleBase
+    {
+        private readonly INoteCreatorService _noteCreatorService;
+        private readonly INoteRemoverService _noteRemoverService;
+        private readonly INoteRetrieverService _noteRetrieverService;
+
+        public NoteModule(INoteCreatorService noteCreatorService, INoteRemoverService noteRemoverService, INoteRetrieverService noteRetrieverService)
+        {
+            _noteCreatorService = noteCreatorService;
+            _noteRemoverService = noteRemoverService;
+            _noteRetrieverService = noteRetrieverService;
+        }
+
+
+        [Command("note add"), Summary("Adds a note to a user")]
+        public async Task CreateNote(IGuildUser user, string message)
```

Comment:
The `message` parameter needs to be marked with the `[Remainder]` attribute, otherwise Discord.Net interprets each word of the message as an additional command parameter and, as a result, this method doesn't get called.